admin/schema.php

```diff
@@ -884,6 +884,11 @@
 
 # Release marker: 1.3.0
 
+$g_upgrade[210] = array( 'AddColumnSQL', array( db_get_table( 'mantis_custom_field_table' ), "
```

Comment:
Should not be `mantis_custom_field_table` but `custom_field`

Reply:
changed to `custom_field`